src/vcpkg-test/binarycaching.cpp

```diff
@@ -229,7 +229,6 @@ TEST_CASE ("generate_nuspec", "[generate_nuspec]")
 {
     auto& fsWrapper = get_real_filesystem();
     VcpkgCmdArguments args = VcpkgCmdArguments::create_from_arg_sequence(nullptr, nullptr);
-    args.imbue_from_environment();
```

Comment:
I can imagine this was needed for the VCPKG_ROOT variable. Now it fails because it couldn't found the vcpkg root.

Reply:
Ah, foo. Thanks for the catch.